api/src/models.d.ts

```diff
@@ -62,3 +62,13 @@ export interface ParticipantActivities {
   principal_id: number;
   completed: boolean;
 }
+
+export interface ParticipantActivitiesWithCompletionStatus {
+  activity_id: number;
+  completed: boolean;
+}
+
+export interface participantAcitivityForProgram {
+  programId: number;
```

Comment:
Every other object member defined in this file uses the underscore version (`created_at`, `updated_at`, etc) and not the camelCase version like you have here. I would suggest changing these to `program_id` and `participant_activities`.